components/dependency_injection/autowiring.rst

```diff
@@ -0,0 +1,399 @@
+.. index::
+    single: DependencyInjection; Autowiring
+
+Using the Autowiring
```

Comment:
I think we should change this title. People who know nothing about "autowiring" will never look for that in Google. Maybe we could change it for *"Defining Services Dependencies Automatically"* or something similar that explains what does "autowiring" solve.